src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="search__bar big__bar"
+    method="post"
+    data-qa="big"
+  >
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      class="search__bar big__bar"
      method="post"
      data-qa="big"
    >
```